decidim-proposals/app/models/decidim/proposals/commentable_proposal.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Decidim
```

Comment:
Wrong file path. This should be under `concerns/`

Reply:
Nice abstraction BTW!

Reply:
@mrcasals can you specify the exact path please? I don't think you mean app/controllers/concerns/..
btw thanks!

Reply:
Same path as `Decidim::HAsReference`, but in `deicim-participatory_processes`

Reply:
Ok, done.